queries.sql

```diff
@@ -40,3 +40,25 @@ SELECT COUNT (*) FROM animals WHERE neutered = false;
 SELECT AVG(wieght_kg) FROM animals;
 SELECT MAX(escape_attempts) FROM animals;
 SELECT species, MAX(wieght_kg) FROM animals GROUP BY species;
+
+/* Day 3 Project */
+SELECT name FROM animals A 
+JOIN owners O ON O.id = A.owner_id AND O.full_name = 'Melody Pond';
+
+SELECT * FROM animals A
+JOIN species S ON A.species_id = S.id AND A.species_id = 1;
+
+SELECT owners.full_name, animals.name FROM animals
+FULL JOIN owners ON owners.id = animals.owner_id;
+
+SELECT COUNT(species_id) FROM animals
+FULL JOIN species ON species.id = animals.species_id GROUP BY species_id;
+
+SELECT name FROM animals
+RIGHT JOIN owners ON owners.id = animals.owner_id AND owner_id = 2 AND species_id = 2;
```

Comment:
- [ ] Instead of adding a condition clause for the `species_id` and the `owner_id`, kindly join the tables in question with the rest of the main table and use the name of the species and owner instead of their `id`. As per [this](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=List%20all%20Digimon%20owned%20by%20Jennifer%20Orwell.) requirement, you should use the names of the species and owner and not the `id`.